.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+.stylelintrc.json
+.hintrc
```

Comment:
- Please, kindly remove the `.stylelintrc.json` and `.hintrc` from the `.gitignore` file as these files needs to be visible in your repo.